### ISSUE TITLE
Confirm overwrite on save as dialog

### ISSUE THREAD

**Issue by aFunkyBass** [e-sollier/filedialpy#3]
Hello @e-sollier I'm going to use your library to pick up files to open and save on my project. 

Now when using the save file dialog it would be great to have a prompt to confirm the overwrite of a chosen file. I know that on Zenity there is a --confirm-overwrite flag which works, and I think it's the same on tkInter. Ufortunately I cannot test in on other platforms.

Is there any chance you could try to add this feature?

Thank you.

**Comment by e-sollier** [comment]
Hi @aFunkyBass,
Thanks for your suggestion, I've added this feature in version 1.3.0, with the confirm_overwrite option that has been added to saveFile. It is set to True by default. 
On my ubuntu installation, zenity already prompted me for an overwrite confirmation even without the --confirm-overwrite flag, but this flag will now be added when confirm_overwrite is set to True, to ensure that this confirmation is shown. 

Let me know if this now works as intended for you.
